src/status_im/ui/screens/communities/create.cljs

```diff
@@ -29,14 +30,20 @@
                 :height               crop-size})
 
 (defn pick-pic []
-  (react/show-image-picker
-   #(>evt [::communities/create-field :image (.-path ^js %)])
-   crop-opts))
+  (js/setTimeout
```

Comment:
Would you mind leaving a comment on why we need a timeout? It's not immediately clear to the untrained eye (myself :) )